add a table in marimo to summarize selected corpora (#306)
Princeton-CDH/remarx: CHANGELOG.md

- Add `xquery` directory with instructions and XQuery files for investigating
    & reporting on continuing paragraph and footnote content in MEGA TEI

### Application

> - Add a summary table in `Quote Finder` notebook displaying user-selected original corpora's information

Reviewer: Now applies to both corpora

```suggestion
- Add summary tables in `Quote Finder` to display details for selected input corpora
```
